.circleci/config.yml

```diff
@@ -600,17 +600,18 @@ jobs:
   #      JOBS: Releases
   # -------------------------
   # Publishes new version onto npm
-  # Only works on stable branches when a properly tagged commit is pushed
```

Comment:
This is no longer the case with the addition of nightly builds, right?

Reply:
Correct